src/flows/shared/steps/OrganizeCollection/Editor/SortableStagedNft.tsx

```diff
@@ -60,13 +62,14 @@ function SortableStagedNft({ nftRef, size }: Props) {
       <StagedNftImage
         nftRef={nft}
         size={size}
+        hideLabel={mini}
         setNodeRef={setNodeRef}
         {...attributes}
         {...listeners}
       />
       <StyledUnstageButton id={id} />
-      <StyledGradient type="top" direction="up" />
-      <StyledGradient type="bottom" direction="down" />
+      <StyledGradient type="top" direction="up" height={mini ? 40 : 64} />
+      {mini ? null : <StyledGradient type="bottom" direction="down" />}
```

Comment:
if the image is small:
1) decrease the top gradient
2) remove the bottom gradient, since we won't show a label preview

this is critical because the gradient would otherwise _block_ the user from being able to drag the small image